x-pack/legacy/plugins/remote_clusters/public/app/sections/components/remote_cluster_form/remote_cluster_form.js

```diff
@@ -500,18 +498,62 @@ export class RemoteClusterForm extends Component {
   }
 
   renderErrors = () => {
-    const { areErrorsVisible } = this.state;
+    const {
+      areErrorsVisible,
+      fields: {
+        name,
+      },
+      fieldsErrors: {
+        name: errorClusterName,
+      },
+      localSeedErrors,
+      seedInput,
+    } = this.state;
+
     const hasErrors = this.hasErrors();
 
     if (!areErrorsVisible || !hasErrors) {
       return null;
     }
 
+    const errorExplanation = [];
+
+    if (errorClusterName) {
+      errorExplanation.push({
+        key: 'nameExplanation',
+        where: i18n.translate('xpack.remoteClusters.remoteClusterForm.inputNameErrorsScreenReaderExplanation', {
+          defaultMessage: 'The input "Name". You have entered: {name}.',
```

Comment:
I agree that this can be worded better. I think people will first need to know the field that is invalid so they have the context to understand the value. So I would suggest this phrasing:

```
The "Name" field is invalid. You have entered {name}.
```

```
The "Seed nodes" field is invalid. You have entered {seedInput}.
```